go/libraries/doltcore/env/actions/table.go

```diff
@@ -48,6 +48,7 @@ func MoveTablesBetweenRoots(ctx context.Context, tbls []string, src, dest *doltd
 			}
```

Comment:
What if we just split this loop into two: one loop that processes all the drops, followed by another that processes the rest?

Reply:
That seems fine, I'll change it.